### PR TITLE
MM-50247: Action frequencies review

### PR DESCRIPTION
**Pull request by agarciamontoro**
#### Summary
This is the result of the research in [the Action frequencies review document](https://docs.google.com/document/d/1A4gRuJBymamP5cTgYkI-CJV3n4Sj4nF07Zn44seuaks/edit#heading=h.5x0d5h95i329). For more information, please refer to the document.

This change needs to be tested before merging, but wanted to open it early in case it provokes more comments :)

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-50247

### CODE REVIEW

File: loadtest/control/simulcontroller/controller.go [hunk 1]
```diff
@@ -110,131 +110,131 @@ func (c *SimulController) Run() {
...
 		},
 		{
 			run:       openDirectOrGroupChannel,
-			frequency: 2,
+			frequency: 0.7784,
 		},
 		{
 			run:       unreadCheck,
-			frequency: 1.5,
+			frequency: 1.0000,
```
Comment (agarciamontoro) [inline loadtest/control/simulcontroller/controller.go:129]: Still to be decided if we remove it or not. See [this comment in the document](https://docs.google.com/document/d/1A4gRuJBymamP5cTgYkI-CJV3n4Sj4nF07Zn44seuaks/edit?disco=AAAAuVgqbU0).

Comment (agnivade) [reply loadtest/control/simulcontroller/controller.go:129]: Commented. I am fine with `1.0` for now.

**Review by agnivade — APPROVED** [review]
Great stuff. Looking forward to a new load test with this!

**Review by streamer45 — APPROVED** [review]
Great work! Looking forward to looking at the the results of these.

**Comment by agarciamontoro** [comment]
I did a first unbounded test with Postgres (comparing v7.9 with v7.10) using the new frequencies: http://3.95.199.228:3000/d/cT9yc0E4k/comparison-0-postgresql-unbounded?orgId=1 Everything looks good!
I want to do a full-fledged performance comparison (both Postgres and MySQL, both bounded and unbounded tests), but this first smoke test shows things look pretty normal.

**Comment by agnivade** [comment]
Nice! The link doesn't seem to open for me. But are we seeing higher number of users with the updated frequencies? I'd expect so.

**Comment by streamer45** [comment]
@agarciamontoro Would it make sense to do a check against Community Grafana at some point to verify there are no significant (e.g. order of magnitude) discrepancies?

**Comment by agarciamontoro** [comment]
@streamer45: Yes, that's something I want to do as well. Testing around 400-500 users for a couple of hours and do a sanity check.

**Pull request closed without merging**